.github/CODEOWNERS

```diff
@@ -328,38 +328,38 @@
 
 
 # Lesson Analytics team.
-/core/controllers/features*.py @brianrodri
-/core/controllers/improvements*.py @brianrodri
-/core/domain/action_registry*.py @brianrodri
-/core/domain/calculation_registry*.py @brianrodri
-/core/domain/event_services*.py @brianrodri
-/core/domain/improvements_domain*.py @brianrodri
-/core/domain/improvements_services*.py @brianrodri
-/core/domain/playthrough_issue_registry*.py @brianrodri
-/core/domain/stats*.py @brianrodri
-/core/domain/visualization_registry*.py @brianrodri
-/core/templates/components/common-layout-directives/common-elements/answer-content-modal.controller*.ts @brianrodri
-/core/templates/components/common-layout-directives/common-elements/answer-content-modal.template.html @brianrodri
-/core/templates/components/common-layout-directives/common-elements/confirmation-modal.template.html @brianrodri
-/core/templates/components/statistics-directives/ @brianrodri
-/core/templates/domain/statistics/ @brianrodri
-/core/templates/domain/improvements/ @brianrodri
-/core/templates/pages/exploration-editor-page/improvements-tab/ @brianrodri
-/core/templates/pages/exploration-editor-page/statistics-tab/ @brianrodri
-/core/templates/services/exploration-improvements.service*.ts @brianrodri
-/core/templates/services/exploration-improvements-backend-api.service*.ts @brianrodri
-/core/templates/services/exploration-improvements-task-registry.service*.ts @brianrodri
-/core/templates/services/exploration-stats.service*.ts @brianrodri
-/core/templates/services/exploration-stats-backend-api.service*.ts @brianrodri
-/core/templates/services/improvements.service*.ts @brianrodri
-/core/templates/services/messenger.service.ts @brianrodri
-/core/templates/services/playthrough*.ts @brianrodri
-/core/templates/services/state-interaction-stats.service*.ts @brianrodri
-/core/templates/services/state-top-answers-stats*.ts @brianrodri
-/extensions/actions/ @brianrodri
-/extensions/answer_summarizers/ @brianrodri
-/extensions/issues/ @brianrodri
-/extensions/visualizations/ @brianrodri
+/core/controllers/features*.py @seanlip @BenHenning
```

Comment:
Let's have all these be just me (@seanlip only)

Reply:
Done.